### DISCUSSION TITLE
Error while  running coremark benchmark as workload on gem5

### DISCUSSION THREAD

**Discussion by waseemabbas-10xe** [gem5/gem5#1131] [Q&A]
While i am running the coremark benchmark as workload on gem5  I am facing this issue if any one has solution please let me know.
![image](https://github.com/gem5/gem5/assets/142871238/2a2c4c03-482f-4a6e-b5af-9c6b9290626b)

**Comment by robhau** [comment]
Duplicate of https://github.com/orgs/gem5/discussions/1132
